### PR TITLE
Auth menu improvements

### PR DESCRIPTION
**Pull request by macitege**

### CODE REVIEW

**Comment by rogerdunnhawaii** [comment]
looks fine

**Comment by MaxxKowalik** [comment]
looks good

File: index.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
               <form id="sign-in">
                 <h4>Login</h4>
-                <input type="email" name="credentials[email]" placeholder="email or username">
+                <input type="text" name="credentials[email]" placeholder="email or username">
```
Comment (discocisco) [inline index.html]: Reminder that we will need to handle the `credentials[email]` into `credentials[username]` when appropriate in our JavaScript files.

**Comment by discocisco** [comment]
I like

File: index.html (hunk 1 above)
Comment (rogerdunnhawaii) [reply index.html]: correct, when the data comes in from getFormFields and we ahve to determine if user inputted email or username in the events.js

File: index.html [hunk 2]
```diff
@@ -74,10 +74,10 @@ <h4>Login</h4>
                         <form id="sign-up">
 <!-- SIGN-UP-FORM -->
                           <h4>Sign Up</h4>
-                          <input type="email" name="credentials[username]" placeholder="username">
```
Comment (MaxxKowalik) [inline index.html]: I've seen the changes, and they were goooooooooood

**Pull request merged**